content/posts/2025-10-02-WildFly38Beta-released.adoc

```diff
@@ -0,0 +1,52 @@
+---
+layout: post
+title:  "New WildFly 38 Beta release"
+aliases: [/news/2025/10/02/WildFly38Beta-released/]
+date:   2025-10-02
+tags:   announcement release
+author: darranl
+description: WildFly 38 Beta is now available for download.
+---
+
+We're excited to announce the release of *WildFly 38 Beta 1*! This version is contains updates, new features, and bug fixes, paving the way for the upcoming final release. This beta version is a crucial step in our development cycle, allowing the community to test and provide feedback on the latest changes.
+
+
+== What's New and Noteworthy?
+
+This release focuses heavily on *component upgrades*, *MicroProfile 7.1 support*, and enhancements to the *WildFly Preview* distribution. Here's a quick rundown of the highlights:
+
+* **Java EE 11 Integration:** A key new feature is the implementation of *Jakarta EE 11 JPA and CDI integration* in WildFly Preview, bringing us closer to full support for the latest Jakarta EE specifications.
+* **MicroProfile 7.1:** WildFly 38 Beta 1 supports the *MicroProfile 7.1* specification, including an update to the latest versions of SmallRye Reactive Messaging and Config. This upgrade ensures that developers can leverage the newest features and improvements in the MicroProfile ecosystem.
+* **Component Upgrades:** We've updated a wide range of dependencies to their latest versions, improving performance, stability, and security. Notable upgrades include:
+ ** **Hibernate ORM 6.6.29.Final** (and 7.1.1.Final in WildFly Preview)
+ ** **Infinispan 15.2.6.Final**
+ ** **Apache Artemis 2.42.0**
+ ** **Netty 4.1.127.Final**, addressing several security vulnerabilities.
+ ** **WildFly Core 30.0.0.Beta3**
+
+* **Quickstart and Documentation Improvements:** We've made significant improvements to our quickstarts and documentation. This includes removing old dependency files, adding new tests, and updating documentation to reflect the latest changes in EE 11 support.
+
+---
+
+== Important Bug Fixes
+
+As a beta release, we've tackled several bugs to ensure a more stable experience:
```

Comment:
Drop "As a beta release". We fix bugs for all release types and betas aren't special.